src/App.vue

```diff
@@ -522,46 +522,15 @@ export default {
       this.setGoodStanding(business.goodStanding)
       this.setHasCourtOrders(business.hasCourtOrders)
 
-      if (business.state === EntityState.HISTORICAL) {
-        await this.parseStateFiling(business.stateFiling) // will throw on error
+      // retrieve stateFiling if the URL is included in the business payload
+      if (business.stateFiling) {
```

Comment:
Retrieve the state filing if it exists instead of only in the case of a historical entity